### ISSUE TITLE
Examples of use?

### ISSUE THREAD

**Issue by prologic** [Spendesk/github-actions-exporter#46]
Does anyone have an example Grafana dashboard(s) they can share or a set of Prometheus alerting rules based on this exporter?

**Comment by AmitBaranes** [comment]
any update here? will save me tons of time

**Comment by vsarunas** [comment]
I assume that everyone keeps on making the same basic dashboards?

**Comment by 95jinhong** [comment]
If you have a dashboard that uses the above exporter, could you share it?

**Comment by SleepyBrett** [comment]
Yeah, I'm currently trying to work out a query that would allow me to alert on the same workflow in the same repo failing more  than some number of times in a row. Even as a seasoned user of prometheus I'm not sure how i'll manage it. Seems like some more metrics could help here.

An all up metric by workflow and repo with a counter for each possible result as noted on github_workflow_run_status would be actually very useful.
